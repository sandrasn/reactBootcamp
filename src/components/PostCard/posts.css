.card {
  margin-top: 15px;
}

.card-title,
.post-text {
  margin-top: 5px;
  padding-left: 5px;
}
