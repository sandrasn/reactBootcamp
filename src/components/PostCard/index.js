import React from "react";
import "./posts.css";

export const sum = (a, b) => a + b;

const PostCard = ({ image, title, text }) => (
  <div className="card">
    <img className="card-img-top" src={image} />
    <h5 className="card-title"> {title}</h5>
    <p className="post-text">{text} </p>
  </div>
);

export default PostCard;
