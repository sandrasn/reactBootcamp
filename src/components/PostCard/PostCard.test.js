import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import PostCard, { sum } from "./index";

describe("testing PostCard", () => {
  const wrapper = shallow(<PostCard />);

  it("sums two numbers", () => {
    expect(sum(3, 5)).toEqual(8);
  });

  it("renders properly", () => {
    expect(wrapper.length).toEqual(1);
  }); //it === test

  it("renders a snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
