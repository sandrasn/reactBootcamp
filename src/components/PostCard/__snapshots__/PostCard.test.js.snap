// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing PostCard renders a snapshot 1`] = `
<div
  className="card"
>
  <img
    className="card-img-top"
  />
  <h5
    className="card-title"
  >
     
  </h5>
  <p
    className="post-text"
  >
     
  </p>
</div>
`;
